Add a test-sources configuration option. (#2062)
pypa/cibuildwheel: docs/options.md

    In configuration files, you can use an array, and the items will be joined with `&&`.

### `CIBW_TEST_SOURCES` {: #test-sources}

> > Files and folders from the source tree that must be copied into the test environment before running the tests.

Reviewer: ```suggestion
> Files and folders from the source tree that are copied into an isolated tree before running the tests
```
